packages/react-router/lib/context.ts

```diff
@@ -58,15 +58,9 @@ export interface RouteMatch<
 
 export interface DataRouteMatch extends RouteMatch<string, DataRouteObject> {}
 
-// Contexts for data routers
-export const DataStaticRouterContext =
-  React.createContext<StaticHandlerContext | null>(null);
-if (__DEV__) {
-  DataStaticRouterContext.displayName = "DataStaticRouterContext";
-}
-
```

Comment:
This was an `UNSAFE` export so ok to remove and add onto `DataRouterContext` (which is also exported as `UNSAFE`)